src/playback/internal/playbackconfiguration.cpp

```diff
@@ -129,6 +129,10 @@ void PlaybackConfiguration::init()
             m_isAuxChannelVisibleChanged.send(idx, val.toBool());
         });
     }
+
+    settings()->valueChanged(PLAY_NOTES_WHEN_EDITING).onReceive(nullptr, [this](const Val&) {
```

Comment:
Nowadays, instead of nullptr, we prefer passing `this`:
```suggestion
    settings()->valueChanged(PLAY_NOTES_WHEN_EDITING).onReceive(this, [this](const Val&) {
```